README.rst

```diff
@@ -573,6 +657,42 @@ different: ``rules`` will ask for the change permission if and only if no rule
 exists for the view permission.
 
 
+Permissions in Django Rest Framework
+------------------------------------
+
+Similar to ``rules.contrib.views.AutoPermissionRequiredMixin``, there is a
+``rules.contrib.rest_framework.AutoPermissionViewSetMixin`` for viewsets in Django
```

Comment:
I didn't want to mix concepts of plain Django (models, views, admin) with those of DRF (viewsets). One might think there are viewsets in Django otherwise. If you don't like it this way, what about `rules.contrib.rest_framework.viewsets`, although I find it a bit too long?

Reply:
Makes sense -- `rules.contrib.rest_framework` might actually be clearly what this is about, so let's keep it as it is.